Rivet/Rivet.psm1

```diff
@@ -3,7 +3,7 @@ $Connection = New-Object Data.SqlClient.SqlConnection
 
 $RivetSchemaName = 'rivet'
 $RivetMigrationsTableName = 'Migrations'
-$RivetMigrationsTableFullName = '{0}.{1}' -f $RivetSchemaName,$RivetMigrationsTableName
+$RivetMigrationsTableFullName = "[$($RivetSchemaName)].[$($RivetMigrationsTableName)]"

```

Comment:
It _might_ be possible to avoid the $(..) wrapping here since bounded with ].

Reply:
I like using the `$()` every time so I don't have to worry about when I need to or not.